bokehjs/src/lib/models/glyphs/step.ts

```diff
@@ -83,6 +95,45 @@ export class StepView extends XYGlyphView {
     ctx.stroke()
   }
 
+  protected _hit_point(geometry: PointGeometry): Selection {
+    /* Check if the point geometry hits this line glyph and return an object
+    that describes the hit result:
+      Args:
+        * geometry (object): object with the following keys
+          * sx (float): screen x coordinate of the point
+          * sy (float): screen y coordinate of the point
+          * type (str): type of geometry (in this case it's a point)
+      Output:
+        Object with the following keys:
+          * 0d (bool): whether the point hits the glyph or not
+          * 1d (array(int)): array with the indices hit by the point
+    */
+    const result = hittest.create_empty_hit_test_result()
+    const point = {x: geometry.sx, y: geometry.sy}
+    const threshold = Math.max(2, this.visuals.line.line_width.value() / 2)
+
+    let x1: number, x2: number
+    let y1: number, y2: number
+    let p0 = {x: NaN, y: NaN}
+    for (let i = 0, end = this.sx.length-1; i < end; i++) {
+      [[x1, y1], [x2, y2]] = this._get_points(i, this.sx, this.sy)
```

Comment:
Given you don't reassign those, you could just put `const` in front of this line and remove declarations above.